tests/Unit/Impl/StreamxClientIntegrationTest.php

```diff
@@ -53,9 +53,11 @@ public static function setUpBeforeClass(): void {
 
     private static function skipTestsIfStreamxIsNotAvailable(): void {
         try {
-            self::$publisher->fetchSchema();
+            if (!self::$publisher->isIngestionServiceAvailable()) {
+                self::markTestSkipped('Skipping test because StreamX is not available');
+            }
```

Comment:
Switch to calling the new method to decide if integration tests should be skipped or executed